OpenSSL/SSL.py

```diff
@@ -1883,6 +1883,18 @@ def get_cipher_version(self):
             return version.decode("utf-8")
 
 
+    def get_protocol_version(self):
+        """
+        Obtain the protocol version of the current connection.
+
+        :returns: The TLS version of the current connection, for example
+            the value for TLS 1.2 would be 0x303.
+        :rtype: :py:class:`unicode`
+        """
+        version = _ffi.string(_lib.SSL_get_version(self._ssl))
+        return version.decode("utf-8")
```

Comment:
If I understand your argument correctly, you are saying that when you query for TLS version you should receive something like 0x769 as a response?  The whole python3 bytes v. unicode thing is still new to me...  What was the thinking behind trying to stick with bytes?

Plenty of things in SSL.py are unicode strings though, including much of the related cipher information.  Should a developer really have to get out their OpenSSL decoder ring everytime she wants to query a cipher?  I think ``TLS_RSA_WITH_AES_256_CBC_SHA256`` makes a lot more sense than ``0x3d``, as long as the upstream doesn't change the formatting of those strings.